plugins/woocommerce/client/legacy/js/frontend/single-product.js

```diff
@@ -307,8 +309,10 @@ jQuery( function( $ ) {
 		e.preventDefault();
 
 		var pswpElement = $( '.pswp' )[0],
-			items       = this.getGalleryItems(),
-			eventTarget = $( e.target ),
+			items         = this.getGalleryItems(),
+			eventTarget   = $( e.target ),
+			currentTarget = e.currentTarget,
+			self				  = this,
```

Comment:
Looks like spacing is a little off here

Reply:
Thanks for catching that, @dkotter .
On VSCode it looked correct 🤔
![image](https://github.com/user-attachments/assets/d71a2a2a-7a94-4153-8283-ddb040012c6c)

Fixed on: [Remove unnecessary white sopace](https://github.com/woocommerce/woocommerce/pull/50730/commits/6e85035d58f88174ca49bc4b2cb2bef78294040f)